models/shopify__customer_cohorts.sql

```diff
@@ -103,10 +103,10 @@ with calendar as (
         windows.order_count_in_month,
         windows.total_price_in_month,
         windows.line_item_count_in_month,
-        backfill_lifetime_sums.previous_cohort_month_number + windows.cohort_month_number as cohort_month_number,
-        backfill_lifetime_sums.previous_total_price_lifetime + windows.total_price_lifetime as total_price_lifetime,
-        backfill_lifetime_sums.previous_order_count_lifetime + windows.order_count_lifetime as order_count_lifetime,
-        backfill_lifetime_sums.previous_line_item_count_lifetime + windows.line_item_count_lifetime as line_item_count_lifetime,
+        coalesce(backfill_lifetime_sums.previous_cohort_month_number, 0) + windows.cohort_month_number as cohort_month_number,
```

Comment:
Do you think we should coalesce ` windows.cohort_month_number` in the rarer (but possible) case that this value returns null on an incremental run? There could be an incremental period where no orders are brought in during the window and thus the partition is run on an empty table.

Reply:
Definitely a rare case, but I think there's no harm adding a `coalesce` statement to maintain data quality :) I've tested locally by running `dbt run` with `coalesce(windows. cohort_month_number)` and I have the same result. Let me add that in my next commit, along with the CHANGELOG and version bump